### DISCUSSION TITLE
Genmon directory not found!!

### DISCUSSION THREAD

**Discussion by SuperC188** [jgyates/genmon#1205] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).
- [X] I have included the generator make and model in my submission.

### Version

Debian Lite Bookworm 
Pi zero 2w

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have submitted logs

### What is/are your question(s)?

 
Hi, I am trying to set up a new pi zero with the gemon program and I am following the instructions. The last script I ran was this,  (git clone https://github.com/jgyates/genmon.git), but I cannot load the directory once I'm done.

The error says:   -bash: cd: genmon: No such file or directory 

What am I doing wrong?

**Comment by jgyates** [comment]
Is your pi connected to the internet? You must have internet access for the pi to clone the repository. What is the output of the git clone command?

**Comment by SuperC188** [comment]
It says it completed the clone, but it also looks like it broke the connection pre maturely. It says it was expecting more data but it also shows 100% loaded and compressed so I can’t tell if the clone was actually completed or not. I have ran it several times, but it says the exact same thing each time.Once done if I type CD to change the directory to genmon, it says the directory is not found.On Oct 25, 2024, at 9:38 PM, jgyates ***@***.***> wrote:﻿
Is you pi connected to the internet? You must have internet access for the pi to clone the repository. What is the output of the git clone command?

**Reply by skipfire** [reply]
Are you using a keyboard and monitor on the Pi, or SSH to execute the commands?  Being able to see the input and output would be helpful.

**Comment by SuperC188** [comment]
Yes, I will see if can get a screenshot.On Oct 26, 2024, at 6:02 AM, Jon R. Helms ***@***.***> wrote:﻿
Are you using a keyboard and monitor on the Pi, or SSH to execute the commands?  Being able to see the input and output would be helpful.

**Comment by SuperC188** [comment]
On Oct 26, 2024, at 6:10 AM, Carlos Williams ***@***.***> wrote:﻿Yes, I will see if can get a screenshot.On Oct 26, 2024, at 6:02 AM, Jon R. Helms ***@***.***> wrote:﻿
Are you using a keyboard and monitor on the Pi, or SSH to execute the commands?  Being able to see the input and output would be helpful.

**Reply by skipfire** [reply]
@SuperC188 if you replied with a picture attached, GitHub doesn't pull the pictures in from email.  You need to visit this thread in a browser and upload the image in a reply.

**Reply by SuperC188** [reply]
![image](https://github.com/user-attachments/assets/17d6fde7-a8cc-4afa-ba0b-a3b04fcfa34f)

**Reply by skipfire** [reply]
@SuperC188 what kind of internet connection do you have?  A quick google search has most of the comments being about slow or unstable internet connections.

https://stackoverflow.com/questions/59282476/error-rpc-failed-curl-92-http-2-stream-0-was-not-closed-cleanly-protocol-erro

In several spots 2 different commands are mentioned as helping with it and it sounds like many have success with either or both.  

Increase the buffer size: `git config --global http.postBuffer 157286400`
Rollback the HTTP version for GIT: `git config --global http.version HTTP/1.1`

**Reply by jgyates** [reply]
bad wifi signal to the pi zero would cause the bad internet. What is the signal strength on the wifi as read from the pi zero?

Also, why are you getting system messages to your console output? that is not the default behavior for ssh. How are you connecting to the pi?

**Reply by SuperC188** [reply, marked as answer]
Connected via USB. Issue resolved, it was a timeout error. After repeated attempts the download finally went through entirely. If for any reason there was a delay in the download it will severe the connection early.

**Reply by SuperC188** [reply]
Any idea what this means in red?
![IMG_2624](https://github.com/user-attachments/assets/e09179ce-e34f-40e4-9d38-1c42755ec98a)

**Reply by jgyates** [reply]
My guess is that this is related to poor internet connectivity. The install script attempted to download a file and did a check to make sure the file was downloaded correctly and the check failed.

**Comment by SuperC188** [comment]
Ok, it seems that the wifi connection may be a little weak. I will continue with the install to see how things work out. Everything seems to be progressing along just fine.On Oct 26, 2024, at 11:01 AM, jgyates ***@***.***> wrote:﻿
My guess is that this is related to poor internet connectivity. The install script attempted to download a file and did a check to make sure the file was downloaded correctly and the check failed.

**Reply by jgyates** [reply]
The this was the only error, you *might* be ok after the install, depending on if you use an add on programs. You can always run the install again with a stronger wifi signal. The bigger issue will probably be spotty access to the web interface or problems notifying of  generator events.

**Reply by SuperC188** [reply]
Well close, but can't access pi via .local in windows browser. Ping comes back ok, but ssh says host id change warning.

**Reply by SuperC188** [reply]
Web app refused to connect.

**Reply by jgyates** [reply]
https://kinsta.com/knowledgebase/warning-remote-host-identification-has-changed/

Also, please provide the exact addresses and commands you are using. This prevents missed details that could help solve your problem. Pictures are always good.
